contracts/epochs/FlowEpoch.cdc

```diff
@@ -429,6 +465,7 @@ pub contract FlowEpoch {
                                                 startView: self.epochMetadata[self.currentEpochCounter]!.endView + UInt64(1),
                                                 endView: self.epochMetadata[self.currentEpochCounter]!.endView + self.configurableMetadata.numViewsInEpoch,
                                                 stakingEndView: self.epochMetadata[self.currentEpochCounter]!.endView + self.configurableMetadata.numViewsInStakingAuction,
+                                                totalRewards: 0.0 as UFix64,
```

Comment:
Should this be 0?

Reply:
Since the value can't be fully accurately calculated until the end of the epoch, I just set it as zero as a placeholder value and update it at the end of the epoch. Do you think we should do a rough calculation here also? Or set it to `nil` instead of `0.0`?

Reply:
Right that makes sense. The part l'm missing is where it's updated at the end of an epoch. I see we set it when [creating the first epoch](https://github.com/onflow/flow-core-contracts/pull/132/files#diff-1da2c6b9c85ee2833c92c917dd4f57655a9f505ff07e2d8088977b6bcdb1969bR701) and when [resetting an epoch](https://github.com/onflow/flow-core-contracts/pull/132/files#diff-1da2c6b9c85ee2833c92c917dd4f57655a9f505ff07e2d8088977b6bcdb1969bR378), but not sure when we'd set `totalRewards` to a non-zero value for epochs started with `startNewEpoch`

Reply:
You're right, that should have been included in the payRewards function. I'll add that and add a test for it